readme.md

```diff
@@ -229,20 +229,38 @@ function Component() {
 
 Reducing nested structures is tiresome. Have you tried [immer](https://github.com/mweststrate/immer)?
 
+Without immer:
+```jsx
+const useStore = create(set => ({
+  lush: { forrest: { contains: { a: "bear" } } },
+  setAnimal: animal => set(state => ({ 
+    ...state, 
+    lush: { 
+      ...state.lush, 
+      forrest: { 
+        ...state.lush.forrest, 
+         contains: {
+           ...state.lush.forrest.contains,
+           a: animal,
+         },
+       },
+     },   
+   })),
+}))
+```
+
+With immer:
 ```jsx
 import produce from 'immer'
 
 const useStore = create(set => ({
   lush: { forrest: { contains: { a: "bear" } } },
-  set: fn => set(produce(fn)),
+  setAnimal: animal => set(produce(state => { state.lush.forrest.contains.a = animal })),
 }))
-
-const set = useStore(state => state.set)
-set(state => {
-  state.lush.forrest.contains = null
-})
 ```
 
+To make it even more comfortable, look at the Middleware paragraph below.
```

Comment:
I think it's good have a sentence here, but if we revert the above, what should we say here?